strapi/strapi
# doc: add diagrams and explanations for DTS contributor doc

### What does it do?

Added diagrams and explanations for 
-  DTS high level architecture
- Stream lifecycle 
- Web socket connection

### Why is it needed?

documentation enhancement

### How to test it?

- cd strapi/docs
- yarn build
- yarn start
- check docs on `http://localhost:3000/`

### Related issue(s)/PR(s)

DX-954
DX-957
